cli/command/container/run.go

```diff
@@ -118,8 +118,6 @@ func runRun(ctx context.Context, dockerCli command.Cli, flags *pflag.FlagSet, ro
 
 //nolint:gocyclo
 func runContainer(ctx context.Context, dockerCli command.Cli, runOpts *runOptions, copts *containerOptions, containerCfg *containerConfig) error {
-	ctx = context.WithoutCancel(ctx)
-
```

Comment:
Don't we still need the non-cancellable context for the `attachContainer` call?

Otherwise, it seems like we're reverting https://github.com/docker/cli/commit/991b1303daec833d662ec54002d1b34c8805afab

Reply:
no, we're not completely reverting this commit. See this line - https://github.com/docker/cli/pull/5645/commits/258f52c409d0041494976cf3a263d2047123f96d#diff-9f722157bd3206a95a2f882edb65882f91b1b07cd5903374d49dad1702eaa8c7R157

I'm essentially moving the `context.WithoutCancel` down.

Reply:
Oh right! For some reason I didn't notice that `WithoutCancel` inside 🙈